src/beast/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ project (Beast)
 set_property (GLOBAL PROPERTY USE_FOLDERS ON)
 
 if (MSVC)
+    # /wd4244 /wd4127
```

Comment:
This comment is somewhat cryptic: I see `/wd4244` in the command line before, but I don't see `/wd4127` and I'm not sure what, if anything, I'm supposed to do.